core/src/scene/drawRule.h

```diff
@@ -70,6 +70,8 @@ struct DrawRule {
     // 480 (on 32bit arch) or 980 byte for params array.
     std::bitset<StyleParamKeySize> active = { 0 };
 
+    bool unified = false;
+
```

Comment:
Why is it necessary to have the unified flag as part of the drawrule? Can't one check in ```PointStyleBuilder::addFeature``` if the drawrule containts text elements?